docs/apis-clients/operate-api/index.md

```diff
@@ -31,8 +31,22 @@ To authorize in the cloud using a JWT token, take the steps in the following exa
 **Example:**
 
 1. Obtain a token to access the REST API.
-   You need `client_id`, `client_secret`, `audience`, and the URL of the authorization server. For more information on how to get these for Camunda Platform 8, look
-   at [Manage API Clients](/docs/components/console/manage-clusters/manage-api-clients/).
+
+   When you create an Operate [client](https://docs.camunda.io/docs/guides/setup-client-connection-credentials/), you get all the information needed to connect to Operate.
+
+   The following settings are needed:
+
+   | Name                     | Description                                     | Default value        |
+   | ------------------------ | ----------------------------------------------- | -------------------- |
+   | client id                | Name of your registered client                  | -                    |
+   | client secret            | Password for your registered client             | -                    |
+   | audience                 | Permission name; if not given use default value | `operate.camunda.io` |
+   | authorization server url | Token issuer server                             | -                    |
+
+For more information on how to get these values for Camunda Platform 8, look
```

Comment:
The markdown table for some reason causes prettier to reset the indentation to zero. This results in, IMO, confusing alignment issues. 

As recommended by @akeller, I'll follow up with @christinaausley on this outside of this PR. 

![image](https://user-images.githubusercontent.com/1627089/216654138-59ebf1d4-00e9-4120-ac53-095af54d7a10.png)

Reply:
Happy to assist as this mildly irritates me as well 😅 Feel free to pop some time on my calendar or we can discuss async next week!